app/src/main/java/com/cmput301f20t21/bookfriends/ui/login/LoginActivity.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * LoginActivity.java
+ * Version: 1.0
+ * Date: October 18, 2020
+ * Copyright (c) 2020. Book Friends Team
+ * All rights reserved.
+ * github URL: https://github.com/CMPUT301F20T21/Book_Friends
+ */
+
+package com.cmput301f20t21.bookfriends.ui.login;
+
+import android.os.Bundle;
+import android.text.Editable;
+import android.view.View;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.cmput301f20t21.bookfriends.R;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+
+/**
+ * Activity that will display the login page and handle user inputs
+ */
+public class LoginActivity extends AppCompatActivity {
+    private TextInputLayout usernameLayout;
+    private TextInputLayout passwordLayout;
```

Comment:
We need an email field as well. Firebase only supports create user by email.

Reply:
ken said he prefer non-email username, do we need email to do authentication? I think for now we can just use query for username and checking for password for authentication. Using other API requires more work(which can be a lower priority task later on)

Reply:
yeah we need email for authentication. Firebase has its own auth api which needs email. For username, we can make a new user name collection that has email as key and username as value.
Firebase already provides auth using email and password, doing secured password authentication ourselves will be redundant.